themes/default/content/blog/walkthrough-of-adopting-infrastructure-as-code/index.md

```diff
@@ -0,0 +1,276 @@
+---
+title: "A Walkthrough of Adopting Infrastructure as Code"
+canonical_url: https://thenewstack.io/a-walkthrough-of-adopting-infrastructure-as-code/
+date: 2023-11-07T19:54:55Z
+draft: false
+meta_desc: A look at the basics of how to get going with Infrastructure as Code, and then a dive into some advanced use cases to show what you can do from there.
+meta_image: meta.png
+authors:
+    - joe-duffy
+tags:
+    - infrastructure-as-code
+    - fundamentals
+
+---
+
+*This is the second of a three-part series originally published on The New Stack.*
+
+Following the first piece in this series, [Infrastructure as Code in Any Programming Language](/blog/infrastructure-as-code-in-any-programming-language/), this walkthrough will show what it takes to get up and running with Infrastructure as Code. Everything we show will be done with Pulumi’s free and open source Infrastructure as Code SDK, which you can [download here](/docs/install/). You can also sign up for Pulumi Cloud [here](https://app.pulumi.com/signup). After discussing the basics of how to get going, we’ll then dive into some advanced use cases to show what you can do from there.
```

Comment:
I'm assuming this is intended to be a faithful repost of the original TNS article (i.e., no content changes). If not, I'd recommend rewording some of the links to avoid the use of "here" in the hyperlinks.

Reply:
I think we can make some simple edits. What do you suggest we say instead of "here"?

Reply:
```suggestion
Following the first piece in this series, [Infrastructure as Code in Any Programming Language](/blog/infrastructure-as-code-in-any-programming-language/), this walkthrough will show what it takes to get up and running with Infrastructure as Code. Everything we show will be done with Pulumi’s [free and open source Infrastructure as Code SDK](/docs/install/). You can also sign up for [Pulumi Cloud](https://app.pulumi.com/signup). After discussing the basics of how to get going, we’ll then dive into some advanced use cases to show what you can do from there.
```